v7/base-austere/templates/base_footer.tmpl

```diff
@@ -0,0 +1,10 @@
+## -*- coding: utf-8 -*-
```

Comment:
What did you change here? Avoid duplicating templates if you only delete the namespace (which I agree is unnecessary, perhaps we should to that upstream)